### DISCUSSION TITLE
Proposal: Get gleam available in official Debian repositories

### DISCUSSION THREAD

**Discussion by dahlbaek** [gleam-lang/gleam#3052] [Ideas & suggestions]
Looking at the install docs https://gleam.run/getting-started/installing/, I notice that Void Linux has Gleam available through the official repositories. It made me wonder: Are there any known blockers to getting Gleam available in the official Debian (and thus Ubuntu) deb repositories, so one could install gleam with a simple `apt install gleam`? And if someone was to volunteer to (try to) make it happen, would that be appreciated or not?

**Comment by lpil** [comment]
Would be cool but we don't maintain any packages so this would be a discussion for the Debian maintainers rather than the Gleam maintainers.

**Comment by dahlbaek** [comment]
I've opened a few prs with a little bit of initial work:
- https://github.com/gleam-lang/gleam/pull/3159
- https://github.com/gleam-lang/gleam/pull/3160
- https://github.com/gleam-lang/gleam/pull/3161
